pkg/operator/controllers/checker/const.go

```diff
@@ -4,5 +4,6 @@ package checker
 // Licensed under the Apache License 2.0.
 
 const (
-	machineSetsNamespace = "openshift-machine-api"
+	azureCredentialSecretNamespace = "kube-system"
+	azureCredentialSecretName      = "azure-credentials"
```

Comment:
@cadenmarchese these constants are not used anywhere, which is why the go linting failed. they should be removed, or used somewhere if they are needed.

```
pkg/operator/controllers/checker/const.go:7:2: `azureCredentialSecretNamespace` is unused (deadcode)
	azureCredentialSecretNamespace = "kube-system"
	^
pkg/operator/controllers/checker/const.go:8:2: `azureCredentialSecretName` is unused (deadcode)
	azureCredentialSecretName      = "azure-credentials"
	^
```

if you run `make lint-go` locally it will fail there as well.

Reply:
You can delete this file.

Reply:
Thanks!